### PR TITLE
Add miner reward in the coinbase transaction

### PR DESCRIPTION
**Pull request by velzevur**
[Pivotal story](https://www.pivotaltracker.com/story/show/155740802)

Currently the knowledge for the amount of the miner reward in the coinbase transaction is present only in the code. This should be verifiable outside the node. Since this will be a subject to governance - it will change so we must provide a mechanism for getting the miner reward at a certain height.

This PR adds the reward as being part of the coinbase transaction itself.

### CODE REVIEW

**Comment by velzevur** [comment]
I'll add a checking of the reward

File: apps/aecore/src/aec_coinbase_tx.erl [hunk 1]
```diff
@@ -15,14 +15,16 @@
...
+         reward/1]).
 
 -behavior(aetx).
 
 -include("common.hrl").
 
 -record(coinbase_tx, {account       = <<>> :: pubkey(),
-                      block_height  :: non_neg_integer()}).
+                      block_height  :: non_neg_integer(),
+                      reward        :: non_neg_integer()}).
```
Comment (lucafavatella) [inline apps/aecore/src/aec_coinbase_tx.erl]: Please integrate check function.

**Comment by lucafavatella** [comment]
Looks fine for the rest.

**Comment by velzevur** [comment]
* Check function is modified to verify the expected reward

* Protocol version is bumped

* Suggested changes in release notes

File: docs/release-notes/RELEASE-NOTES-0.9.0.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 [This release][this-release] is focused on TODOFILLMEIN.
 It:
 * Improves the stability of the testnet.
+* Eases verification of the coinbase transaction, by including the minin reward in it. This impacts both the consensus and the persisted DB;
```
Comment (lucafavatella) [inline docs/release-notes/RELEASE-NOTES-0.9.0.md]: Typo minin mining

File: docs/release-notes/RELEASE-NOTES-0.9.0.md [hunk 3]
```diff
@@ -3,6 +3,7 @@
 [This release][this-release] is focused on TODOFILLMEIN.
 It:
 * Improves the stability of the testnet.
+* Eases verification of the coinbase transaction, by including the mining reward in it. This impacts both the consensus and the persisted DB;
```
Comment (lucafavatella) [inline docs/release-notes/RELEASE-NOTES-0.9.0.md]: ok

**Pull request merged**